src/main/java/core/basesyntax/Mail.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+import core.basesyntax.service.Inventory;
+import core.basesyntax.service.InventoryFromCsv;
+import core.basesyntax.service.Report;
+import core.basesyntax.service.ReportToCsv;
+import core.basesyntax.storage.Storage;
+
+public class Mail {
+    public static void main(String[] args) {
+        Storage storage = new Storage();
+
+        Inventory inventory = new InventoryFromCsv();

```

Comment:
Avoid hardcoding file names outside of the Main class. Consider passing the file name as a parameter to the `InventoryFromCsv` class or using a configuration file to manage such constants .